apps/charterafrica/package.json

```diff
@@ -61,6 +61,7 @@
     "nodemon": "^2.0.20",
     "payload": "^1.6.4",
     "prop-types": "^15.8.1",
+    "qs": "^6.11.0",
```

Comment:
Do we need more than what [URLSearchParams](https://nodejs.org/docs/latest-v16.x/api/url.html#class-urlsearchparams) provides?